tests/lib/nested.sh

```diff
@@ -617,7 +617,18 @@ nested_start_core_vm_unit() {
     PARAM_CPU=""
     PARAM_TRACE="-d cpu_reset"
     PARAM_LOG="-D $NESTED_LOGS_DIR/qemu.log"
-    PARAM_SERIAL="-serial file:${NESTED_LOGS_DIR}/serial.log"
+    # Open port 7777 on the host so that failures in the nested VM (e.g. to
+    # create users) can be debugged interactively via
+    # "telnet localhost 7777". Also keeps the logs
+    #
+    # XXX: should serial just be logged to stdout so that we just need
+    #      to "journalctl -u nested-vm" to see what is going on ?
```

Comment:
then we need to handle the fact that for whatever reason when I tried this journald's default config loses some of the output when we turn on more verbose logging, so journald on the host VM needs to be configured differently I think to not lose earlier logs

perhaps the solution is simpler but I couldn't figure out why `journalctl -ef -u nested-vm` did not show the full log of the boot when I changed it to just `-serial mon:stdio`